centraldogma/src/test/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplierIntegrationTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.centraldogma;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import com.linecorp.centraldogma.client.CentralDogma;
+import com.linecorp.centraldogma.common.Change;
+import com.linecorp.centraldogma.common.Entry;
+import com.linecorp.centraldogma.common.Query;
+import com.linecorp.centraldogma.common.Revision;
+import com.linecorp.centraldogma.internal.Jackson;
+import com.linecorp.centraldogma.testing.junit4.CentralDogmaRule;
+import com.linecorp.decaton.processor.runtime.Property;
+import com.linecorp.decaton.processor.runtime.PropertyDefinition;
+
+public class CentralDogmaPropertySupplierIntegrationTest {
+
+    @Rule
+    public CentralDogmaRule centralDogmaRule = new CentralDogmaRule();
+
+    private static final String PROJECT_NAME = "unit-test";
+    private static final String REPOSITORY_NAME = "repo";
+    private static final String FILENAME = "/subscription.json";
+
+    private static final PropertyDefinition<Long> LONG_PROPERTY =
+            PropertyDefinition.define("num.property", Long.class, 0L,
+                                      v -> v instanceof Long && (Long) v >= 0L);
+
+    @Test(timeout = 50000)
+    public void testCDIntegration() throws InterruptedException {
+        CentralDogma client = centralDogmaRule.client();
+
+        final String ORIGINAL =
+                "{\n"
+                + "  \"num.property\": 10,\n"
+                + "  \"ignore.keys\": [\n"
+                + "    \"123456\",\n"
+                + "    \"79797979\"\n"
+                + "  ],\n"
+                + "  \"partition.processing.rate\": 50\n"
```

Comment:
[nits] I know these are just dummy properties, but let's align `ignore.keys` and `partition.processing.rate` to actual Decaton's property name (i.e. `decaton.ignore.keys` and `decaton.processing.rate.per.partition`) to avoid unnecessary confusion?

Reply:
Ah, realized that it's taken from original CentralDogmaPropertySupplierTest.... lol
So would you mind fixing about this?

Reply:
sure